pkg/test/framework/components/echo/common/deployment/echos.go

```diff
@@ -68,6 +68,9 @@ type Config struct {
 	// Custom echo instances will be accessible from the `All` field in the namespace(s) under which they
 	// were created.
 	Configs echo.ConfigGetter
+
+	// ServiceNamePrefix allows setting a common prefix for all Services in the Configs
+	ServiceNamePrefix string
```

Comment:
What's the need for this?

Reply:
It's there so that I can deploy multiple sets of echo services in the same namespaces - to test that overlapping namespaces actually work to the extent that you can mix workloads from different meshes in the same namespace (and that they can't talk to each other)

Reply:
I admit it's a bit ugly but I don't think we have another kind of 'test workload' I could use as secondary workloads

Reply:
Ah, yeah that makes sense. Could you maybe add a comment indicating it can be used this way? Or maybe just comment in your test body that this is how you're using it? Might be helpful for others down the line

Reply:
done! thanks for taking a look